app/src/main/java/org/wikipedia/main/MainFragment.kt

```diff
@@ -123,6 +125,12 @@ class MainFragment : Fragment(), BackPressedHandler, MenuProvider, FeedFragment.
         }
     }
 
+    private val onboardingLauncher = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
```

Comment:
Naming: can we name this something much more specific than `onboardingLauncher`? Maybe `activityTabOnboardingLauncher`?